sentry-failure/src/lib.rs

```diff
@@ -49,6 +50,33 @@ fn fail_typename<F: Fail + ?Sized>(f: &F) -> (Option<String>, String) {
     }
 }
 
+#[derive(Default)]
+pub struct FailureIntegration;
+
+impl Integration for FailureIntegration {
+    fn name(&self) -> &'static str {
+        "failure"
+    }
+
+    fn setup(&self, cfg: &mut ClientOptions) {
+        cfg.in_app_exclude.push("failure::");
+        cfg.extra_border_frames.extend_from_slice(&[
+            "failure::error_message::err_msg",
+            "failure::backtrace::Backtrace::new",
+            "failure::backtrace::internal::InternalBacktrace::new",
+            "failure::Fail::context",
+        ]);
+    }
+}
+
+pub fn panic_extractor(info: &PanicInfo<'_>) -> Option<Event<'static>> {
```

Comment:
For this method, a specific note that this is useful with the `PanicIntegration` would be great, along with some usage examples in a doc comment.